torch/csrc/shim_common.cpp

```diff
@@ -578,3 +586,34 @@ torch_get_mutable_data_ptr(AtenTensorHandle tensor, void** ret_data_ptr) {
     *ret_data_ptr = t->mutable_data_ptr();
   });
 }
+
+AOTI_TORCH_EXPORT AOTITorchError
+torch_new_string_handle(const char* data, size_t length, StringHandle* handle) {
+  AOTI_TORCH_CONVERT_EXCEPTION_TO_ERROR_CODE({
+    auto str_ptr = std::make_unique<std::string>(data, length);
+    *handle = reinterpret_cast<StringHandle>(str_ptr.release());
+  });
+}
+
+AOTI_TORCH_EXPORT AOTITorchError torch_delete_string(StringHandle handle) {
+  AOTI_TORCH_CONVERT_EXCEPTION_TO_ERROR_CODE({
+    auto str_ptr = reinterpret_cast<std::string*>(handle);
+    delete str_ptr;
```

Comment:
If you created it as `unique_ptr<>` you should delete as a unique_ptr.
If you want to use delete here, you need to use new above right?